src/changelog/2023/12/email-alerting-node-red-crash.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Email Alerts for Audit Log Events
+title: "FlowFuse update: Now receive email alerts for Audit Log events, ensuring timely notifications for Node-RED instance crashes."
```

Comment:
I think this should be description and not title?

Reply:
Oh, sorry. I've corrected it now.